FParsecCS/FParsecCS-NETCore.csproj

```diff
@@ -0,0 +1,23 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <AssemblyName>FParsecCS</AssemblyName>
+    <RootNamespace>FParsec</RootNamespace>
+    <TargetFramework>netstandard2.0</TargetFramework>
```

Comment:
Depending on reach, you may want to multitarget: https://docs.microsoft.com/en-us/dotnet/standard/library-guidance/cross-platform-targeting#multi-targeting

Basically, we can only guarantee that consumers on .NET Framework 4.7.2 can reliably use .NET Standard 2.0 components. It's our collective bad for that 😢

Since FSharp.Core is built against .NET Framework 4.5 as its lowest version, this may be a reasonable target:

`<TargetFrameworks>netstandard2.0; net45</TargetFrameworks>`